### PR TITLE
EOS23627 - Implement ldifParser and remove data hardcoding

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
Data was hardcoded/pasted inside py script. As  a result- code was not looking clean 

# Design
 
Used ldif package from python-ldap and instead of hardcoding data , read the data from ldifs directly

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]:  Y
-  Confirm All CODACY errors are resolved [Y/N]:  Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]:  
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [x] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by ujjwalpl — APPROVED** [review]

**Review by bkirunge7 — APPROVED** [review]

File: py-utils/src/utils/setup/openldap/iam-constraints.ldif [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/iam-constraints.ldif:2]: 2020 -> 2021

File: py-utils/src/utils/setup/openldap/ppolicy-default.ldif [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/ppolicy-default.ldif:2]: 2020 -> 2021

File: py-utils/src/utils/setup/openldap/ppolicymodule.ldif [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/ppolicymodule.ldif:2]: 2020 -> 2021

File: py-utils/src/utils/setup/openldap/ppolicyoverlay.ldif [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/ppolicyoverlay.ldif:2]: 2020 -> 2021

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Minor comments

File: py-utils/src/utils/setup/openldap/ppolicyoverlay.ldif (hunk 4 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/ppolicyoverlay.ldif:2]: Fixed

File: py-utils/src/utils/setup/openldap/ppolicymodule.ldif (hunk 3 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/ppolicymodule.ldif:2]: Fixed

File: py-utils/src/utils/setup/openldap/ppolicy-default.ldif (hunk 2 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/ppolicy-default.ldif:2]: Fixed

File: py-utils/src/utils/setup/openldap/iam-constraints.ldif (hunk 1 above)
Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/iam-constraints.ldif:2]: Fixed

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**